elixir/lib/cloner.ex

```diff
@@ -30,7 +29,11 @@ defmodule GithubRepoCloner.Cloner do
     |> run_command
   end
 
+  defp run_command("") do
+    {:error, "No repositories found"}
```

Comment:
Return an `error` tuple if the command is not run. This includes scenarios where no repos are found for the username or page and if the username is invalid.